ssl/statem/statem_local.h

```diff
@@ -96,7 +96,7 @@ WORK_STATE ossl_statem_server_post_process_message(SSL *s, WORK_STATE wst);
 __owur int tls_get_message_header(SSL *s, int *mt);
 __owur int tls_get_message_body(SSL *s, size_t *len);
 __owur int dtls_get_message(SSL *s, int *mt);
-__owur int dtls_get_message_body(SSL *s, size_t *len);
+__owur int ossl_dtls_get_message_body(SSL *s, size_t *len);
```

Comment:
I find it questionable to only change one of these. Indeed, why don't we rename all of the functions declared here?
*After all, functions declared here are all internal*, it's not like anyone outside this project depend on the names here remaining the same!

Reply:
Picking just one *is* silly

Reply:
which is why I didnt do this :)

Reply:
What's the definition of our public API?

Reply:
> What's the definition of our public API?

How hard can it be?  If it's declared by our *public headers* (`<openssl/whatever.h>`), it's public.  Anything else is internal.

(note that "public" and "extern" isn't the same thing.  We have internal functions that are extern, and are therefore exposed in the static library, but thankfully not in the shared library)

Reply:
So, every function declared in these internal headers are *internal*, and should therefore be fine to rename.  That would make them internally consistent, instead of some starting with `ossl_` and others not.

Reply:
OTC: this symbol change to be reverted.